tests/specs/mcd/dsvalue-read-pass-summarize-spec.k

```diff
@@ -0,0 +1,142 @@
+// #Not( { 255 &Int #lookup( ACCT_STORAGE , 1 ) /Int 1461501637330902918203684832716283019655932542976 #Equals 0 } )
+// #And <generatedTop>
+//   <kevm>
+//     <k>
+//           #halt
+//       ~> _DotVar2
+//     </k>
+//     <schedule>
+//       ISTANBUL
+//     </schedule>
+//     <ethereum>
+//       <evm>
+//         <output>
+//           #buf( 32 , #lookup( ACCT_STORAGE , 2 ) )
+//         </output>
+//         <statusCode>
+//           EVMC SUCCESS
+//         </statusCode>
+//         <endPC>
+//           284
+//         </endPC>
+//         <callState>
+//           <program> DSValue_bin_runtime </program>
+//           <jumpDests> #computeValidJumpDests(DSValue_bin_runtime) </jumpDests>
+//           <id>
+//             ACCT_ID
+//           </id>
+//           <callData>
+//             b"W\xde&\xa4" ++ CD
+//           </callData>
+//           <callValue>
+//             0
+//           </callValue>
+//           <wordStack>
+//             1474176676 : .WordStack
+//           </wordStack>
+//           <localMem>
+//             b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x80" [ 128 := #buf( 32 , #lookup( ACCT_STORAGE , 2 ) ) ]
+//           </localMem>
+//           <pc>
+//             284
+//           </pc>
+//           <gas>
+//             #gas( VGas +Int -2043 )
+//           </gas>
+//           <memoryUsed>
+//             5
+//           </memoryUsed>
+//           ...
+//         </callState>
+//         ...
+//       </evm>
+//       <network>
+//         <accounts>
+//           AccountCellMapItem( <acctID>
+//             ACCT_ID
+//           </acctID> , <account>
+//             <acctID>
+//               ACCT_ID
+//             </acctID>
+//             <storage>
+//               ACCT_STORAGE
+//             </storage>
+//             ...
+//           </account> ) _DotVar5
+//         </accounts>
+//         ...
+//       </network>
+//     </ethereum>
+//     ...
+//   </kevm>
+//   ...
+// </generatedTop>
+// #And { false #Equals <acctID>
+//   ACCT_ID
+// </acctID> in _DotVar5 keys( ) }
+
+// Converted to summary
+// 313s
+    claim [DSValue.read.pass]:
+      <k> #execute => #halt ... </k>
+      <schedule> ISTANBUL </schedule>
+      <evm>
+        <output> _ => #buf(32, #lookup(ACCT_STORAGE, 2)) </output>              // from counterexample
+        <statusCode> _ => EVMC_SUCCESS </statusCode>
+        <callState>
+          <id> ACCT_ID </id>
+          <program> DSValue_bin_runtime </program>
+          <jumpDests> #computeValidJumpDests(DSValue_bin_runtime) </jumpDests>
+          <callData> #abiCallData("read", .TypedArgs) ++ CD => ?_ </callData>
```

Comment:
I'm getting a warning about `CD` being defined but not used. What does it stand for here?

Reply:
In EVM execution starts with the `<callData>` buffer (byte-array) filled with the arguments to call with. `#abiCallData("read", .TypedArgs)` is the needed bytes to tell the EVM code to "execute the `read()` function", but it's perfectly legal for you to have any junk bytes after that first 4 bytes. So we put `_++ CD` to show that the proof works in the context of having any arbitrary call-data after those bytes.

Reply:
But we could replace `CD` with `_CD` to make the warning go away.

Reply:
```suggestion
          <callData> #abiCallData("read", .TypedArgs) ++ _CD => ?_ </callData>
```